pkg/apis/rke.cattle.io/v1/rkecontrolplane_types.go

```diff
@@ -0,0 +1,180 @@
+package v1
+
+import (
+	"github.com/rancher/wrangler/v3/pkg/genericcondition"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type RKEControlPlaneSpec struct {
+	ClusterConfiguration `json:",inline"`
+
+	// AgentEnvVars is a list of environment variables that will be set on
+	// the cluster agent deployment and system agent service.
+	// +optional
+	AgentEnvVars []EnvVar `json:"agentEnvVars,omitempty"`
+
+	// LocalClusterAuthEndpoint is the configuration for the local cluster
+	// auth endpoint.
+	// +optional
+	// +nullable
```

Comment:
For this field, the previous CRD didn't have this as `nullable` (wrangler didn't turn structs into nullable automatically), so we might not need this.

(Note that most of the other fields in this struct were nullable before, and won't be anymore).

Reply:
I'll probably reach out to you tomorrow (today?) and try to go through one by one which fields should remain `nullable`, and which ones we might be able to safely remove from.